### PR TITLE
exec user command by server

### PR DESCRIPTION
**Pull request by ZimGil**
The server will now execute the given command in the given folder.

In this solution the server keeps the last given folder as the destination folder to work on.
We can easily make a default folder for cases where no folder is given.

There is still no implementation of any kind of input verification

Solves #31

### CODE REVIEW

File: server/server.js [hunk 1]
```diff
@@ -25,9 +25,25 @@ app.get('/', function(req, res) {
 });
 
 app.post('/command', function(req, res) {
-  console.log('\n########################################################################################\n');
-  const keys = Object.keys(req.body);
-  keys.forEach( key => console.log(key, req.body[key]));
+  if (req.body.folder) {       // to make sure folder is received
```
Comment (ortichon) [inline server/server.js]: remove comment

File: server/server.js [hunk 2]
```diff
@@ -25,9 +25,25 @@ app.get('/', function(req, res) {
...
 
 app.post('/command', function(req, res) {
-  console.log('\n########################################################################################\n');
-  const keys = Object.keys(req.body);
-  keys.forEach( key => console.log(key, req.body[key]));
+  if (req.body.folder) {       // to make sure folder is received 
+    process.chdir(req.body.folder);
+  }
+
+  commandEvent = childProcess.exec(req.body.command, (err, stdout, stderr) => {
```
Comment (ortichon) [inline server/server.js]: use `const`  / `let`

File: server/server.js [hunk 3]
```diff
@@ -25,9 +25,25 @@ app.get('/', function(req, res) {
...
+    if (err) {
+      console.log(err);
+      return;
+    }
+  });
+
+  commandEvent.stdout.on('data', function(data) {
```
Comment (ortichon) [inline server/server.js]: we should decide if we use ES5 annonimous function style:
`function() {`
or ES6 function style:
`()=> {`

File: server/server.js [hunk 4]
```diff
@@ -25,9 +25,25 @@ app.get('/', function(req, res) {
...
-  console.log('\n########################################################################################\n');
-  const keys = Object.keys(req.body);
-  keys.forEach( key => console.log(key, req.body[key]));
+  if (req.body.folder) {       // to make sure folder is received 
+    process.chdir(req.body.folder);
+  }
+
+  commandEvent = childProcess.exec(req.body.command, (err, stdout, stderr) => {
+    if (err) {
+      console.log(err);
```
Comment (ortichon) [inline server/server.js]: consider using `console.error()`

File: server/server.js (hunk 3 above)
Comment (ZimGil) [reply server/server.js]: We'll go with ES6

**Pull request merged**